src/ape/pytest/runners.py

```diff
@@ -95,8 +95,13 @@ def pytest_runtest_setup(self, item):
             # isolation is disabled via cmdline option
             return
 
-        # list of scopes for each fixture of the test (including `autouse` fixtures)
-        scopes = [item._fixtureinfo.name2fixturedefs[f][0].scope for f in item.fixturenames]
+        fixture_map = item.session._fixturemanager._arg2fixturedefs
+        scopes = []
+        for name, fixture in fixture_map.items():
+            if name in item.fixturenames:
+                definitions = fixture_map[name]
+                for definition in definitions:
+                    scopes.append(definition.scope)
```

Comment:
Might be a little faster and easier to read
```suggestion
        scopes = [
            definition.scope
            for name, definitions in fixture_map.items()
            if name in item.fixturenames
            for definition in definitions
        ]
```